src/app/components/search/Search.test.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { WrapperComponent } from "../../../tests/test-utils";
-import Search from "./Search.jsx";
+import Search from "./Search";
```

Comment:
Is this meant to be here? You haven't mentioned it in your description

Reply:
yes, it is lint clean up. I don't think we need explaining every single lint change. react components do not need extension

Reply:
In the PR template the 'What' can easily include "linter improvements"

Reply:
I think it is implicit

Reply:
It's only implicit if it relates to the files you are working in. If there's improvements in other areas it's worth noting to the person who's picking up your review that you have run the linter in a file outside of your PR to avoid doubt and this relatively pointless conversation